src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -759,7 +759,7 @@
 "Reset_Subtitle" = "Reset App";
 
 /* Safari */
-"safari_corona_website" = "https://www.bundesregierung.de/corona-warn-app-faq";
+"safari_corona_website" = "https://www.bundesregierung.de/breg-de/themen/corona-warn-app/corona-warn-app-englisch/corona-warn-app-faq-1758636";
```

Comment:
I suggest using the same URL the Android app uses.
```suggestion
"safari_corona_website" = "https://www.bundesregierung.de/corona-warn-app-faq-englisch";
```

Reply:
@BjoernPetersen Great, thanks. That links looks far better than what the CMS provides.